articles/virtual-machines/workloads/oracle/configure-oracle-dataguard.md

```diff
@@ -6,28 +6,31 @@ ms.service: virtual-machines
 ms.subservice: oracle
 ms.collection: linux
 ms.topic: article
-ms.date: 08/02/2018
-ms.author: kegorman
+ms.date: 03/23/2023
+ms.author: goblackw
 
 ---
 
+
 # Implement Oracle Data Guard on an Azure Linux virtual machine 
 
 **Applies to:** :heavy_check_mark: Linux VMs 
 
-Azure CLI is used to create and manage Azure resources from the command line or in scripts. This article describes how to use Azure CLI to deploy an Oracle Database 12c database from the Azure Marketplace image. This article then shows you, step by step, how to install and configure Data Guard on an Azure virtual machine (VM).
+This article demonstrates, step by step, how to deploy Oracle on an Azure virtual machine.  You will then install and configure Data Guard to replicate a database to a secondary (backup) database on a second Oracle instance.  Azure CLI will be used to create and manage all required Azure resources.  To secure the environment, no ports will be publicly accessible and a Bastion instance will be included to provide access to the Oracle VMs.
 
-Before you start, make sure that Azure CLI is installed. For more information, see the [Azure CLI installation guide](/cli/azure/install-azure-cli).
+> [!Important]
+> 
+> Before you start, verify that Azure CLI is installed. For more information, see the [Azure CLI installation guide](/cli/azure/install-azure-cli).
 
 ## Prepare the environment
```

Comment:
Hi @glbwell0 For H2s we usually add an introductive paragraph (a few sentences) describing what the customer will learn about or do throughout that section, like a high-level summary of all the H3s & describing what to expect. Can you please add one under the H2.